### PR TITLE
Add support to aggregate response data

### PR DESCRIPTION
**Pull request by heathermusson**
Adds a new aggregate_responses table which counts up the number of responses for each answer. There is a new field, `count` on the answer type which has the number of responses this answer has received for the current live session.

### CODE REVIEW

File: app/graphql/fydp_backend_schema.rb [hunk 1]
```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 
 class FydpBackendSchema < GraphQL::Schema
-  use GraphQL::Subscriptions::ActionCableSubscriptions
```
Comment (alexgirard) [inline app/graphql/fydp_backend_schema.rb:4]: why was this removed?

**Review by alexgirard — APPROVED** [review]
🥳

File: app/graphql/fydp_backend_schema.rb (hunk 1 above)
Comment (heathermusson) [reply app/graphql/fydp_backend_schema.rb:4]: There was an error. This is included elsewhere, which I did not realize.
